styles.css

```diff
@@ -251,7 +257,8 @@ input[type="text"] {
   row-gap: 10px;
   column-gap: 10px;;
   /*margin-right:20%;*/
-  
+  height: 100%;
+  overflow-y: scroll;
```

Comment:
the game_list now has height = `.content` height (calculated above). and scroll the list if overflow